### PR TITLE
Corrected tolerance to have the unit of force as defined in the OpenMM doc

### PR DESCRIPTION
**Pull request by ziyuanzhao2000**
This is just to fix the issue mentioned here #1446 in the showcase notebook.

### CODE REVIEW

**Review by j-wags — APPROVED** [review]
Excellent - Thanks for the quick PR @ziyuanzhao2000! As far as I can tell, this is the only place where we provide the tolerance to OpenMM's minimizer, so this should be the only change that's needed. It is a bit concerning that I don't know how OpenMM was interpreting the previous values... and in fact these examples CI runs are taking 5-10 minutes longer than usual...  I'll tinker around with this a bit before merging. 

Half the CI is failing because your PR is coming from a fork, so it can't access some org secrets - That's our problem, not yours :-) The half of the CI that is passing shows that your change works!

I've updated the releasenotes and will merge this once I've resolved the timing thing.

**Comment by j-wags** [comment]
I'm unable to reproduce the timing difference locally, so I'll just assume that this is running slower because the CI machines are busy on a Monday morning :-)

**Comment by ziyuanzhao2000** [comment]
Thanks for your prompt feedback! Glad to hear that there's no timing difference for your local run. I do agree it's a bit concerning how OpenMM handled the previous unit and didn't throw up an error.

**Pull request merged**
